pkg/services/action.go

```diff
@@ -109,13 +109,16 @@ func NewSubmitSystemIntake(
 			return err
 		}
 
+		// Send SystemIntake to CEDAR Intake API
 		intake.SubmittedAt = &updatedTime
 		alfabetID, validateAndSubmitErr := validateAndSubmit(ctx, intake)
 		if validateAndSubmitErr != nil {
-			return validateAndSubmitErr
+			appcontext.ZLogger(ctx).Error("Submission to CEDAR failed", zap.Error(validateAndSubmitErr))
+		} else {
+			intake.AlfabetID = null.StringFrom(alfabetID)
```

Comment:
Maybe add a comment/TODO/NOTE here explaining the nuance we talked about in the meeting, that intakes can have null values for two reasons (failed as is the case here and legacy SharePoint intakes)

Reply:
I like this. Adding!

Reply:
71a74b3e

Reply:
63f01499